central/rbac/utils/namespace_permission_evaluator.go

```diff
@@ -47,17 +47,26 @@ func (c *namespacePermissionEvaluator) RolesForSubject(ctx context.Context, subj
 
 func (c *namespacePermissionEvaluator) getBindingsAndRoles(ctx context.Context, subject *storage.Subject) ([]*storage.K8SRoleBinding, []*storage.K8SRole) {
 	q := search.NewQueryBuilder().
+		AddStringsHighlighted(search.RoleID, search.WildcardString).
+		AddBoolsHighlighted(search.ClusterRole, true).
+		AddBoolsHighlighted(search.ClusterRole, false).
```

Comment:
I want to be sure that this clause behaves correctly, in that it means cluster role with any values (true or false), and records with any cluster role value are returned.

Reply:
Added one additional test case to cover this.